src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -108,33 +122,143 @@ def __init__(
         max_batch_size: int,
         max_batch_time: int,
     ) -> None:
-        self.__commit = commit
-        self.__max_batch_size = max_batch_size
-        self.__max_batch_time = timedelta(milliseconds=max_batch_time)
-        self.__messages_since_last_commit = 0
-        self.__last_commit = datetime.now()
-        self.__ready_to_commit: MutableMapping[Partition, Position] = {}
-        self.__closed = False
+        self._closed: bool = False
```

Comment:
I removed the leading double underscore and stuck to a single one. There's no reason to have two of them, and having a single one makes debugging and testing way easier.

Reply:
There is a reason to have two underscores and you should keep it. Double underscore will get mangled so it will be impossible to access from outside. Single underscore is just a convention but it is still essentially public.

Reply:
It prevents accidental use of these "private" values from outside which is generally good - this is precisely the same reason that you find it more difficult to test.

Reply:
> It prevents accidental use of these "private" values from outside

Actually, it doesn't. Double-leading underscore doesn't actually make a variable private, and one can still access it from the outside even if it's a bit harder. This means the person coding has the responsibility of correctly using the internal variables a class has. A single leading variable means that variable is internal, so it's really the responsibility of the person coding how/if something should be used.

The only (afaik) exception to that is conflicting variables with the same name in parent classes. However, that problem doesn't exist in this case, and the usage of one or the other becomes a matter of style (afaik there isn't one yet) or personal preference.